### PR TITLE
chore(dedicated): Fixes DAR 453: customers can't enable tracing

### PR DESCRIPTION
**Pull request by jstirnaman**
Updates Cloud Dedicated docs to reflect current status of https://github.com/influxdata/EAR/issues/5677. Fixes https://github.com/influxdata/DAR/issues/453.
  - Move trace to draft status
  - Remove trace-id from code sample
  - Use troubleshoot.md as alias for /influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/trace/
  - Add warning with note to contact support
  - Reorganize troubleshooting sections

Closes #

_Describe your proposed changes here._

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/optimize-queries.md [hunk 1]
```diff
@@ -87,11 +87,3 @@ If the table contains 10 columns, the difference in performance between the
...
 In a table with over 1000 columns, the `SELECT *` query is slower and
 less efficient.
-
-## Analyze and troubleshoot queries
-
-Use the following tools to analyze and troubleshoot queries and find performance bottlenecks:
-
-- [Analyze a query plan](/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/analyze-query-plan/)
-- [Enable trace logging for a query](/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/trace/)
-- [Retrieve `system.queries` information for a query](/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/system-information/)
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/optimize-queries.md:97]: Should we still leave "Analyze a query plan" and "Retrieve `system.queries` information for a query? Dedicated users should still be able to do these.

**Review by sanderson — APPROVED** [review]
Looks good. Just one question.

File: content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/optimize-queries.md (hunk 1 above)
Comment (jstirnaman) [reply content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/optimize-queries.md:97]: > Should we still leave "Analyze a query plan" and "Retrieve `system.queries` information for a query? Dedicated users should still be able to do these.

I kept them, just moved them to `troubleshoot.md`. Overall, though, I'm still not happy with the structure/organization in `troubleshoot-and-optimize`.

Comment (jstirnaman) [reply content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/optimize-queries.md:97]: @sanderson It's a little better now, I think. I'll need to port some structure changes to Clustered and Serverless.

**Review by sanderson — APPROVED** [review]
🚀

File: content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/trace.md [hunk 2]
```diff
@@ -12,8 +12,11 @@ related:
   - /influxdb/cloud-dedicated/query-data/sql/
   - /influxdb/cloud-dedicated/query-data/influxql/
   - /influxdb/cloud-dedicated/reference/client-libraries/v3/
+draft: true
```
Comment (appletreeisyellow) [inline content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/trace.md:15]: Does this mean this page is hidden to public now?

**Review by appletreeisyellow — APPROVED** [review]
Looks good to me. thank you @jstirnaman

**Pull request merged**